src/test/resources/liquibase/harness/change/changelogs/redshift/addColumn.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Aren't these the same as in the default folder in test-harness? We need to override input changelog files only if they don't work for the platform, usually with an explanation of why it's needed and why defaults don't work